cartographer/transform/transform_interpolation_buffer.cc

```diff
@@ -59,23 +59,7 @@ transform::Rigid3d TransformInterpolationBuffer::Lookup(
         return timestamped_transform.time < time;
       });
   const auto end = start;
-  if (end->time == time) {
-    return end->transform;
-  }
-  --start;
```

Comment:
We should keep this code, right? Add a unit test? I think this is for the case that `timestamped_transforms_.size() == 1`.

Reply:
done